tools/scripts/aducm.mk

```diff
@@ -81,20 +81,18 @@ DFP_FILES = $(call rwildcard,$(DFP_DRIVERS),*.c)
 #Not for aducm3029
 DFP_IGNORED_FILES += $(DFP_DRIVERS)/flash/adi_flash_data.c $(DFP_DRIVERS)/rtc/adi_rtc_data.c
 
-ADUCM_SRCS += $(filter-out $(DFP_IGNORED_FILES), $(DFP_FILES))
+PLATFORM_SRCS += $(filter-out $(DFP_IGNORED_FILES), $(DFP_FILES))
```

Comment:
the changes in this commit should be sqashed with the previous commit, they both modify aducm.mk regarding PLATFORM_SRCS

Reply:
The previous one is about INCS and reordering them. There was a bug with the order.